h2o-py/h2o/estimators/model_selection.py

```diff
@@ -325,6 +326,10 @@ def __init__(self,
                excluded in the dataset.
                Defaults to ``None``.
         :type influence: Literal["dfbetas"], optional
+        :param multinode_mode: For maxrsweep only.  If enabled will attempt to perform sweeping action using multiple
+               nodes inthe cluster.  Defaults to false.
```

Comment:
```
:param multinode_mode: For maxrsweep only.  If enabled will attempt to perform sweeping action using multiple
               nodes __inthe__ cluster.  Defaults to false.
```
needs regenerating the client code before merging. `ModelSelectionV3` contains the correct version.

Reply:
Thank you tomas.  Fixed.